locales/ja-JP/messages.lang.php

```diff
@@ -277,7 +268,7 @@
 $GLOBALS['__l10n']["Loaded"]="読み込み完了";
 $GLOBALS['__l10n']["Loaded at Startup"]="起動時に実行";
 $GLOBALS['__l10n']["Loading"]="読み込み中";
-$GLOBALS['__l10n']["Log in"]="ログイン";
+$GLOBALS['__l10n']["Log in"]="ログインID";
```

Comment:
'ID' has been added any reason?

Reply:
yes, because of https://github.com/oat-sa/tao-core/blob/master/locales/ja-JP/messages.po#L653. I'm not able to say if it's an error or not